includes/helper-functions.php

```diff
@@ -182,10 +182,7 @@ function edac_is_gutenberg_active() {
 	$gutenberg    = false;
 	$block_editor = false;
 
-	if ( has_filter( 'replace_editor', 'gutenberg_init' ) ) {
-		// Gutenberg is installed and activated.
-		$gutenberg = true;
-	}
+	$gutenberg = has_filter( 'replace_editor', 'gutenberg_init' );
```

Comment:
Ideally we'd check if Gutenberg is active using a check like the one below:
```php
$gutenberg = defined( 'IS_GUTENBERG_PLUGIN' );
```

That's something we can do in a followup PR when we rething a bit how we check for plugins 👍